components/search/search.tsx

```diff
@@ -78,6 +79,23 @@ const Search: React.FC<Props> = (props: Props): JSX.Element => {
         new SearchUserProvider(actions.autocompleteUsersInTeam),
     ]);
 
+    const handleKeyDown = (e: KeyboardEvent) => {
+        if (Utils.cmdOrCtrlPressed(e) && e.key === 'f' && !isSideBarRight) {
+            if (isDesktopApp() || (!isDesktopApp() && e.shiftKey)) {
+                handleAddSearchTerm(`In:${currentChannelName} `);
```

Comment:
it is a bit misleading, but the variable `currentChannelName` can possibly be undefined if I am not mistaken.
At least that is what can be derived from the props-mapping:

`currentChannelName: getCurrentChannel(state)?.name,`

The nullish coalescent operator is what kind of leaves a question mark here.

Reply:
First of all, I see what you did there :P
Second, I'm pretty sure `getCurrentChannel` only returns null before the app is fully loaded, so it was more of a protection to make sure that the component loaded correctly and didn't throw an error. By the time the app is loaded it should always be defined.